shippo/resource.py

```diff
@@ -421,6 +421,27 @@ def create(cls, api_key=None, **params):
         """
         return super(Track, cls).create(api_key, **params)
 
+    @classmethod
+    def create_webhook(cls, api_key=None, **params):
+        """Create a Webhook to push events from Shippo (i.e tracking,transations)
+
+        Arguments:
+            **params
+                url (str) -- url of your webhook (make sure it is not behind basic auth.
+                                  endpoint must return 200 when it receives a POST  
+                event (str) -- any valid webhook event as listed here https://goshippo.com/docs/webhooks.
+                is_test (str) -- set the webhook object to test or live mode
+        Keyword Arguments:
+            api_key (str) -- an api key, if not specified here it will default to the key
+                             set in your environment var or by shippo.api_key = "..."
+
+        Returns:
+            (ShippoObject) -- The server response
+        """
+
+        return super(Track, cls).create(api_key, **params)
```

Comment:
This doesn't look like it would work, but the tracking example you have prints out the expected response?

Reply:
@hwangm you're right it wasn't going to work.  I ended up adding a class method for the webhook creation, listing, and removal.